committee-steering/sig-governance.md

```diff
@@ -0,0 +1,629 @@
+an oncall rotation must be setup containing SIG ~~Foo~~ members.  The
+purpose of this group is to proactively monitor signals for the health
+of the areas owned by the SIG, and to provide an escalation point for
+urgent or critical issues.  If no oncall group is chosen, this group
+will default to the technical lead group.
+
+**Note:** Oncall calendar infrastructure for th community is
+forthcoming.
+
+#### Test [SLO][slo]
+
+SIG ~~Foo~~ has an objective of fixing any broken test within 2 days
+either by addressing the underlying issue, or rolling back the breaking
+change.  The oncall member has the power to freeze code merges for
+SIG ~~Foo~~ areas if tests remain broken for more than 2 days.
+
+#### Flaky tests
+
+Flaky tests are defined at having >10% failure rate by job.
+Tests observed to
+be consistently flaky will be prioritized and fixed within 1 release
+cycle.  If no members of the SIG can commit to fixing the flaky
+test within a release single, then Tech leads have the power to
+stop accepting PRs for new features until staffing is committed.
+
+#### Test escalation
+
+If assets owned by SIG ~~Foo~~ are suspected to be breaking other
+SIGs tests or making their tests flaky, then members of those
+SIGs may send an email to the sig mail list and escalate to the test
+oncall member.  If the issue cannot be resolved, the issue may be
+escalated to technical leads.
+
+### Addressing operational or technical blocking issues
+
+If for some reason the SIG is unable to meet its responsibilities
+and this is negatively impacting either the project or SIG,
+then the project lead or technical lead group (whoever is
+responsible for the issue), may enforce that the SIG stop accepting
+contributions for new development, and only accept bug fixes
+and sustaining engineering until the issue has been resolved.
+If no resolution can be found, the issue may be escalated to the
+[Steering Committee][steering-commitee].
+
+---
+
+# FAQ
+
+## Why is it so long
+
+This document attempts to ensure the institutional knowledge
+of individuals managing SIGs today is well documented and available
+to all members in the SIG.  There are actually many responsibilities and
+processes that have developed as part of facilitating management
+of the project by a SIG comprised of a decentralized and diverse
+group of individuals.
+
+## Why so much structure
+
+This document intentionally leans toward more structure rather than
+less.  There are 2 reasons for this:
+
+- It is easy for SIGs to remove or edit sections in this
+  template to reduce structure, whereas it is harder to make up new
+  structure from scratch
+- The SIG governance questionnaire showed a consistent preference for
+  more structure over less.  (almost all answers were to have
+  the same or more amount of structure)
+
+## Why use mailing lists for tracking roles
+
+Using mailing lists to track roles makes it simple to contact
```

Comment:
I don't disagree that there needs to be a way to contact people, but members of mailing lists aren't visible to non-members (or even non-managers).

The canonical lists need to be in github.